### PR TITLE
Art cooldown + better values

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Makes artwork guns no longer have insainly crazy stats for shotguns/sniper/nade/ect
Makes artwork oddities have a bigger range
Gives artwork better odds to be a gun if using maxium inspiration
Removes a dead end error path
Artwork creation perk cooldown to help stop as much spam of artwork
Allows rng fire-delay for artwork guns

### CODE REVIEW

**Comment by PvtStupidhead** [comment]
Sorry, copy-pasting this from the Discord, since I feel like this is a better way place to give feedback.

Just to give some context: I've experienced that using the stopgaps of 40, 55, 70 and 85 insight doesn't have nearly as high of an impact at making stuff specificly that you want as it ought to have. Given that these values are hidden for most new artists, the benefits of using the low-amount one is that if you have a good enough (red) anomaly inside your artist bench, you can beat the bad odds of making statues and crap by spam-making out 40 and regaining your sanity fast enough.

Just like farming in FO13, you beat bad odds by repeating the dice throw over and over and over again in multiple instances, at which point you'll finally roll what you wanted and get the gun, artist mod or oddity (although oddities seems to be glitched out atm, as even having had a super-ultra rare purple one only produced at best 2-3 red stats, and then the rest will be 50/50 of blue and green). 

Having this cooldown, while I understand it'll remove the statue-spam as you witnessed in the pool and stem the amount of money an artist can make, it also crushes up an artists ability to beat the odds and create something noteworthy. An artists gun can't be modified, so removing the modifications on the 19mm, 40mm and 14.5mm anti-rifle is a huge slap in the face for artists because the only benefit to them now (if you roll maximum capacity) is they'll fit in your pocket, nothing more or less, since fire-rate is capped and the weapons cannot be altered by external means.

And with the (justified) nerf to the artists modifications, you'll usually roll really shit ones (200-300% tool degradation anyone?) with almost nothing else of note to them. Even weaponry modifications, the armor penetration "increase" usually degrades the penetration of the gun in question, meaning that one stat is instantly at a much worse odds than the others, and most of the times these modifications get ignored unless I plead with people to use them. While a miner would love and jump at the chance to get 300% work speed, is it worth it when they are usually tainted with increased tool degradation at such a high amount? Or when the damage modifier can only reach 5 for melee hits, would you really want to put it on your best melee weapon? 

The pool incident is a direct result of me applying what I feel is the only truely logical conclusion to the problem of - "How do I get what I want made out of artistry when the odds are against me?" Just roll low, again and again and again, until I eventually get high enough stats or just by sheer luck create what I want. The pool was also emptied around 20 minutes after I showed it off to everyone (and nobody had swung around and seen what I was doing in the midst of its construction). The stats in question for the weaponry make it reach a point where I wouldn't want to create high calibers anymore, only 9mm or 10mm, since they atleast have a chance to roll really damn neat stats. I usually go huge spurts of artistry, and then go about and roleplay with people around me or ask for their services before doing another huge spurt. Now, to get the maximum effect, I would have to jog back and forth, never truely commit myself to talking to people for too long in fear of missing out that 10 minute cooldown. 

I am grateful that the shotgun's stats have been looked at, as it was completely moronic that all forms of ammunition (anything with penetration) was just unusable, and you could at best create a 1:1 damage shotgun.  I am also willing to understand that having a guaranteed 1.4 dmg/pen modifier on the higher calibers was abuseable, but when for instance the 9mm pistol can only have anywhere between 5-7 bullets in it, they must deal good damage to compensate.

If these are the changes you wish to push out onto artists, I understand. But I must say that then it should be the artists decision on what they make, not a dice throw. If I want to make a gun, I will demand a gun and it'll be made. If I want a statue, it'll be a statue, and I can pick the name. If it's supposed to be a gambling machine, I should just like in Las Vegas become a gambling addict and sit there from dawn till dusk till night till dawn rolling that slot machine. Because if I can't, I never have a chance to win big.

File: code/datums/perks/cooldown.dm [hunk 1]
```diff
@@ -50,3 +50,19 @@
 		holder.stats.changeStat(STAT_COG, 5)
 	..()
 
+/datum/perk/cooldown/artist_no
+	name = "Burnout"
+	desc = "Art as taken its toll and you are resting the soul. Tea would help."
```
Comment (PvtStupidhead) [inline code/datums/perks/cooldown.dm:55]: Spelling error, has instead of as

**Comment by benj8560** [comment]
generally not a fan of this. Artist is a underplayed and unloved role we don't really need to bottleneck/punish like this

**Comment by PvtStupidhead** [comment]
Having reviewed these new changes, I can say I'm pleased with most of them giving back atleast some more control, the fire-rate ones and the buff to 9mm capacity is greatly appreciated. Reviewing the code, I just realized that Robustness and Toughness currently have no usage for artistry, maybe just best for now to throw them together for revolvers?

**Comment by PvtStupidhead** [comment]
image

I honestly don't know. When this is the kind of rolls I get pretty frequently, I don't know why I should have a 10 minute cooldown to try again (around 8 if I keep sipping that iced green tea). Is this 1/3rd the value of the stuff the Artificier's Perfection Cube produces? While Artists have no other mechanics they can interact with while waiting on the CD?

**Comment by cdb-is-not-good** [comment]
gonna fullmerge this fastlike due to needing to free up the artist file for another testmerge. Further changes can definitely be made if we feel it necessary.

**Pull request merged**